invenio_openaire/jsonschemas/funder/funder-v1.0.0.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "http://json-schema.org/schema#",
```

Comment:
We should also provide the Elasticsearch mappings, but that can come afterwards. I'll open an issue.